app/routes/users/components/ChangePassword.js

```diff
@@ -48,7 +48,7 @@ const ChangePassword = ({
         type="password"
         component={TextInput.Field}
       />
-      <Button disabled={disabledButton} submit>
+      <Button disabled={disabledButton} submit danger>
```

Comment:
Not very crucial, but I think we should have a consistent order for the props, ie. `danger` always coming before or after `disabled`

Reply:
Sure, it kinda bothered me as well. It's tedious work, but I'll do it.